docs/markdown/foundations/content_standards.md

```diff
@@ -0,0 +1,928 @@
+     - Use “you/your”
+     - Examples: 
+       - Also get an email when someone: Comments on your Pin
+  </Group>
+  <Group>
+  <Dont title="Don't"/>
+     - Don’t use On | Off for toggles
+     - Don’t use “me/mine”
+     - Examples:
+        - Also get an email when someone: Comments on my Pin
+  </Group>
+</TwoCol>
+
+---
+
+### Specific verbs
+
+In UI writing, sometimes it’s hard to choose between two seemingly similar verbs. Here are some tips:
+
+**Pick vs. choose vs. select:**
+Pick and choose sound more conversational in en_US/UK, and should always be used when discussing taste (“Pick your favorite topics!”) or other more human concepts. If you’re thinking about saying “choose,” try “pick” instead (unless you’re trying to avoid monotony). 
+
+*Pick:*
+- Pick 5 topics you love
+
+<br/>
+
+**See vs. view:**
+View derives from some action on the user's part (choosing filters, say) and See comes from work Pinterest’s already done for the user (like providing additional search results/related Pins).
+
+*See:*
+- Come see! (tooltip on more ideas tabs)
+- See ideas trending in Hounds (email notif)
+
+*View:*
+- View your map (map button CTA)
+- View in chronological order (made up filter sort)
+
+---
+
+### Tenses
+
+Use the [present perfect](https://www.grammarly.com/blog/present-perfect-continuous-tense/) tense in UI writing, even when using a contraction seems more conversational. 
+
+<TwoCol>
+  <Group>
+    <Do title="Do"/>
+     - You have a new follower
+  </Group>
+  <Group>
+  <Dont title="Don't"/>
+     - You've got a new follower
+  </Group>
+</TwoCol>
+
+---
+
+### Pin puns
+
+We all love a good Pin pun, but unfortunately, they don’t translate very well. We save Pin puns for internal comms, but don’t use them in external language or content. 
+
+---
+
+## More resources
+
+**Pinterest resources**
+- [Pinterest Content Design Hub](https://coda.io/d/Content-Design-Hub_d3AuBYHVHhZ/Content-Design-Hub_suVcA#_lucnd): A central hub for everything Content Design at Pinterest. 
+- [Pinterest Dictionary](https://docs.google.com/document/d/1JQWpTbnIj3w1lJtZoyYcOieSveOsGjuSeHqMdAfetU4/edit): A glossary of Content Design terminology at Pinterest. 
+- [Pinterest Brand Guidelines](https://brand.pinterest.com/): Standards for brand design at Pinterest. *Always check with the brand design team to make sure you're using the latest guidelines.*
+
+**There’s a world outside of Pinterest?!**
+- [Writing + content strategy resources](https://docs.google.com/document/d/1KIpmqk6peL8C-qAbFcEKBlwCZz3fhRntAJDK7mtsJik/edit): Bay Area-writing classes, articles about writing and more!
+- [The AP Style Guide](https://owl.purdue.edu/owl/subject_specific_writing/journalism_and_journalistic_writing/ap_style.html): When in doubt, refer to…
+- [A Progressive’s Style Guide](https://s3.amazonaws.com/s3.sumofus.org/images/SUMOFUS_PROGRESSIVE-STYLEGUIDE.pdf): When in doubt about how to refer to a specific group of people, refer to…
+
```

Comment:
This page is _huge_ are we sure we don't want to split it up into multiple pages? I think it would help with discoverability of all the sections

Reply:
I do agree that the doc is long. 

@BTaylorPins  - thoughts? Maybe we could do this now. I don't mind breaking it out. 

I was thinking for navigation:

- Content Standards
   - Voice
   - Formatting
   - Grammar and Punctuation
   - Language & Terms
   - UI Elements
   - Syntax & Structure